packages/jest-runtime/src/index.js

```diff
@@ -739,3 +739,5 @@ class Runtime {
 }
 
 module.exports = Runtime;
+// $FlowFixMe
```

Comment:
Currently I'd say no but let's have this discussion another time. I just feel like some stuff is gonna break. We could potentially do it before we tag 15.0.

why not `static transform() { return transform.call(null, arguments); }`?